modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLManagementToolbarPropsTransformer.js

```diff
@@ -234,7 +235,34 @@ export default function propsTransformer({
 	};
 
 	const permissions = () => {
-		const keys = getAllSelectedElements().get('value');
+		const map = new Map();
```

Comment:
Not sure if it happens only to me, but it took me a while to get what this map was for. Do you think it could be good to rename it as `selectedModelClassNameS` or something similiar? This way you clearly get why it has to be just 1 (folder, file entry or shorcut)

Reply:
Sorry, I realised I was wrong here... this map includes modelClassNames and the values. 
Would it make sense to have separate arrays for clarity? Something like: 
```

		const selectedFileEntries = getAllSelectedElements().get('value');

		let selectedModelClassNames = [];

		getAllSelectedElements().each((element) => {
			selectedModelClassNames.push(
				element.getData('modelclassname') ?? defaultModelClassName
			);
		});

		const modelClassName = new Set(selectedModelClassNames);

		if (modelClassName.size > 1) {
			return;
		}
```

Reply:
I prefer to go with the map, for example, if we want to unify files+shorcuts in the same modal in the future. 

I wouldn't say I like the name but I can't think of anything better and I left the @adolfopa's name.